python/boost_hist.py

```diff
@@ -27,43 +27,25 @@ def TH2F(name, title, x_nbins, x_bin_low, x_bin_high, y_nbins, y_bin_low, y_bin_
         b_y_axis_name = title_split[2]
     b_name = title_split[0]
     b_label = name
-    return Hist(
-        hist.axis.Regular(bins=x_nbins, start=x_bin_low, stop=x_bin_high, name=b_x_axis_name),
-        hist.axis.Regular(bins=y_nbins, start=y_bin_low, stop=y_bin_high, name=b_y_axis_name),
-        label=b_label,
+    
+    return hist.dask.Hist(
+        hist.axis.Regular(bins=x_nbins, start=x_bin_low, stop=x_bin_high, name=b_x_axis_name), 
+        hist.axis.Regular(bins=y_nbins, start=y_bin_low, stop=y_bin_high, name=b_y_axis_name), 
+        label=b_label, 
         name=b_name,
         storage=hist.storage.Weight()
         )
 
-
-def TH2F_category(name, title, x_categories, y_nbins, y_bin_low, y_bin_high):
```

Comment:
Why was this dropped?

Reply:
It is still there, you can see it by opening a file in the branch.